processor/processor.go

```diff
@@ -639,7 +641,11 @@ func (proc *HandleT) processJobsForDest(jobList []*jobsdb.JobT, parsedEventList
 							panic(fmt.Errorf("typecast of singularEvent to map[string]interface{} failed"))
 						}
 						shallowEventCopy["message"] = singularEventMap
-						shallowEventCopy["destination"] = reflect.ValueOf(destination).Interface()
+
+						if !misc.ContainsString(customDestinations, destType) {
```

Comment:
Why do we need this?

Reply:
While doing load testing, was getting some error while sending events to transformer - EOF. As while setting up aws MSK, in the doc they asked to store the entire jdk ca certs to store in client truststore. While testing, I put all the ca certs (it was huge) that was there in the truststore to webapp config.  So, while data was being sent to transformer, the size of it was too big. That is why getting the error.
Thought that as we have put text area there in webapp, may be we can get similar error. So, removed it for these destinations as we dont need config in transformer.